packages/plugin-dom-layout/src/DomLayout.ts

```diff
@@ -166,7 +167,12 @@ export class DomLayout<T extends DomLayoutConfig = DomLayoutConfig> extends JWPl
         // We cannot always expect a 'contentEditable' attribute on the main
         // ancestor. So we expect to find the main editor ZoneNode if we are in
         // the editable part of Jabberwock.
-        return node?.ancestor(ZoneNode).managedZones.includes('main') || false;
+        return (
+            (node &&
+                (!this.editor.mode || this.editor.plugins.get(Mode).isEditable(node)) &&
```

Comment:
This branching would not be require is the editor would always have a mode.

Reply:
It would mostly not be required if I had managed to do what I left as a todo. That is precisely the line that is concerned with that todo.

Reply:
I see no TODO.

Reply:
Commit message:
> `TODO: if there is no mode, the `editable` property of the node should be used as default. This is not working properly as of yet.`